torch/csrc/tensor/python_tensor.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include <Python.h>
+#include <ATen/ATen.h>
+
+namespace torch { namespace tensor {
+
+// Initializes the Python tensor type objects: torch.Tensor, torch.FloatTensor,
+// etc. and binds them in their containing modules.
+void initialize_python_bindings(PyObject* module);
+
+// Sets the concrete type constructed by calls to torch.Tensor()
```

Comment:
this also sets the default types for other calls, right (e.g. ones with a dtype parameter)?  Maybe amend the comment to that effect.